python/src/aiconfig/editor/client/src/utils/oboeHelpers.ts

```diff
@@ -0,0 +1,37 @@
+import oboe, { Options } from "oboe";
```

Comment:
this isn't adding functionality to this pr right?

Reply:
It was needed I think on client for Ryan to process. I kept it here from his diff in https://github.com/lastmile-ai/aiconfig/pull/651

cc @rholinshead will let you keep or delete moving forward